drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -360,6 +520,155 @@ mlx5_vdpa_setup_notify_relay(struct vdpa_priv *priv)
 	}
 	return 0;
 }
+static int
+mlx5_vdpa_dma_map(struct vdpa_priv *priv)
+{
```

Comment:
This function needs some inline comment to doc the algorithm, it is not trivial. 

you me and ido know it, but for the handover to R&D it will much help.